test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
Because bitbucket needs the repo owner and the `{1234}` (repo ID) to identify the repo. This is the decision that I discussed with @stjohnjohnson a while ago when implementing this module. You can look at the API here: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D

For github, it doesn't even need the repo owner, so it looks like this: `github.com:1234:mybranch`